Fused syscalls (finalize, utm init, utm alloc) (#1)

alexthomas1 opened the pull request.

dayeol commented:
The submitted changeset is hard to i terpret and review because of different indentation style. Because of that, a lot of irrelevant lines appear in the changeset.
Can you make the indentation 2-space tab and resubmit? I'll also comment on the file to clarify

alexthomas1 commented:
> The submitted changeset is hard to i terpret and review because of different indentation style. Because of that, a lot of irrelevant lines appear in the changeset.
> Can you make the indentation 2-space tab and resubmit? I'll also comment on the file to clarify

Yes, sorry. I switched the two space indenting and changed my auto-formatting to conform a bit to the coding style.
